PyInstaller/__init__.py

```diff
@@ -78,7 +78,9 @@
             pass
 
 
-if is_win:
+if os.environ.get('PYINSTALLERCONFIGDIR'):
```

Comment:
@mplatings I could you use underscores in this variable name like: PYINSTALLER_CONFIG_DIR